app/Helpers/database/code-note.php

```diff
@@ -28,33 +28,7 @@ function getCodeNotesData(int $gameID): array
     return $codeNotesOut;
 }
 
-function getCodeNotes(int $gameID, ?array &$codeNotesOut): bool
```

Comment:
This function was redundant with `getCodeNotesData()`. I deleted it and migrated all usages to `getCodeNotesData()`.